modules/core/src/main/java/org/apache/ignite/IgniteCluster.java

```diff
@@ -593,7 +593,6 @@
      * Set a shutdown policy on a cluster.
      *
      * @param shutdownPolicy Shutdown policy.
-     * @return True if the policy has been set, false otherwise.
      */
     public void shutdownPolicy(ShutdownPolicy policy);
```

Comment:
Javadoc is red: 
@param shutdownPolicy
public void shutdownPolicy(ShutdownPolicy policy);

Variable names differ.